src/Core/Checkout/Cart/Event/AfterLineItemAddedEvent.php

```diff
@@ -0,0 +1,53 @@
+<?php declare(strict_types=1);
```

Comment:
Thank you for updating the naming scheme - since we need this to be consistent though, please change the events to e.g. "BeforeLineItemAddedEvent" and "AfterLineItemAddedEvent". The existing "LineItemAddedEvent" can then be flagged for removal (`@deprecated ...`) with the upcoming major version v6.4.0 (it already is marked as deprecated, due to an interface change).